docs/react-spring/introduction.mdx

```diff
@@ -98,7 +98,10 @@ You can also support this project by becoming a sponsor. Your logo will show up
 ## Gold sponsors
 
 <a href="https://aragon.org/">
-  <img width="300" src="https://wiki.aragon.org/design/logo/svg/imagetype.svg" />
+  <img
+    width="300"
+    src="https://assets.website-files.com/602ced7da46f594dd6700e15/602cf7ea9fc1f41358f0ae27_5e99778310343ed2dfe89331_logo_big.svg"
+  />
```

Comment:
Not sure if this is preferred, but they have more brand assets here: https://aragon.org/brand.